CinemaTicketSalesSystem/CinemaTicketSalesSystem/Views/EditMovie/CreateNewMovie.cshtml

```diff
@@ -0,0 +1,100 @@
+﻿@model CinemaTicketSalesSystem.Models.CreateMovieViewModel
+@{
+    ViewBag.Title = "CreateNewMovie";
+    Layout = "~/Views/Shared/MasterView.cshtml";
+}
+@Scripts.Render("~/bundles/jquery")
```

Comment:
It should not be on each page.
You should create Shared Layout and put it there